pom.xml

```diff
@@ -39,6 +39,8 @@
     <testcontainers.version>1.19.8</testcontainers.version>
     <mockito-inline.version>5.2.0</mockito-inline.version>
     <aws-sdk.version>2.28.21</aws-sdk.version>
+    <k8s-client-java.version>21.0.2</k8s-client-java.version>
+    <com.google.protpbuf.protobuf-java.version>4.27.5</com.google.protpbuf.protobuf-java.version>
```

Comment:
Assuming this is because of a trivy issue

Reply:
yeah, added a note in the trivyignore so we don't forget